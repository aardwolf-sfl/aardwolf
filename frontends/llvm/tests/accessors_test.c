struct foo {
    int n;
};

struct foo global[5];

struct baz {
    struct foo foo;
};

int bar(int i) {
    return global[i].n;
}

int quo() {
    struct baz b;
    return b.foo.n;
}
