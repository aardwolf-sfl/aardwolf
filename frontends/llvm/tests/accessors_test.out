
function: bar

#1 -> #2  ::  defs: %1 / uses:  [@1 11:13-11:13]  { arg }
#2 ->   ::  defs:  / uses: %2[%1].%4 [@1 12:5-12:5]  { ret }

function: quo

#3 ->   ::  defs:  / uses: %3.%4.%4 [@1 17:5-17:5]  { ret }

@1 = tests/accessors_test.c
